### PR TITLE
Add ISV debugger docs

### PR DESCRIPTION
**Pull request by ruthemmanuelle**
### What does this PR do?
Adds a draft of the ISV Customer Debugger docs to the salesforcedx-vscode-apex-debugger README. 

I'll ask Nick to look at this after Gunnar is done reviewing it.

After the tech reviews, the content will need to be sent to the Content Experience team's editor. 

This PR shouldn't be merged until the feature is about to be released.

### What issues does this PR fix or reference?
@W-4691368@

### CODE REVIEW

File: packages/salesforcedx-vscode-apex-debugger/README.md [hunk 1]
```diff
@@ -93,6 +94,44 @@ To auto-complete potential request type values for `"requestTypeFilter"`, press
...
+1. In your subscriber’s org, from Setup, enter **Apex Debugger** in the Quick Find box, then click **Apex Debugger**.
+1. Click **Start Partner Debugging Session**.
+1. In the Using Salesforce Extensions for VS Code section, to copy the `forceide://` URL, click **Copy to Clipboard**.
+1. In VS Code, press Cmd+Shift+P (macOS) or Ctrl+Shift+P (Windows or Linux) to open the command palette, then run **SFDX: Create and Set Up Project for ISV Debugging**.
+1. When directed to, paste the `forceide://` URL into the prompt and then press Enter.
+1. When directed to, enter a name for your debugging project and then press Enter.
```
Comment (guw) [inline packages/salesforcedx-vscode-apex-debugger/README.md]: @ruthemmanuelle by default the subscriber org name will be used as project name. Do you want to add something like "_or accept the default name_"?

File: packages/salesforcedx-vscode-apex-debugger/README.md [hunk 2]
```diff
@@ -93,6 +94,44 @@ To auto-complete potential request type values for `"requestTypeFilter"`, press
...
+1. Choose a location to store the project, and then click **Create Project**.
+1. Wait for the project generation process to finish. VS Code creates a Salesforce DX project that contains your packaged metadata, your subscriber's metadata, and skeleton classes for other packages in the org. VS Code also creates a launch configuration (`launch.json` file) for the project. When the project is ready, VS Code opens it for you in a new window.
```
Comment (guw) [inline packages/salesforcedx-vscode-apex-debugger/README.md]: Would it make sense to add a sentence that the process can be long running (depending on org size) while code and managed packages are retrieved from the subscriber org?

File: packages/salesforcedx-vscode-apex-debugger/README.md [hunk 3]
```diff
@@ -93,6 +94,44 @@ To auto-complete potential request type values for `"requestTypeFilter"`, press
...
+1. In the new window, from the Explorer view, open an Apex class or trigger that you want to set breakpoints in.
+1. Switch to the Debug view.
+1. To set a breakpoint, click the gutter to the left of the line numbers.
```
Comment (guw) [inline packages/salesforcedx-vscode-apex-debugger/README.md]: 117 and 118 seems to be reversed, i.e. I would set breakpoint first and then switch to Debug view.

**Comment by ruthemmanuelle** [comment]
@vazexqi - This is ready for your review.

File: packages/salesforcedx-vscode-apex-debugger/README.md [hunk 4]
```diff
@@ -93,6 +94,44 @@ To auto-complete potential request type values for `"requestTypeFilter"`, press
...
+1. Click **Start Partner Debugging Session**.
+1. In the Using Salesforce Extensions for VS Code section, to copy the `forceide://` URL, click **Copy to Clipboard**.
+1. In VS Code, press Cmd+Shift+P (macOS) or Ctrl+Shift+P (Windows or Linux) to open the command palette, then run **SFDX: Create and Set Up Project for ISV Debugging**.
+1. When directed to, paste the `forceide://` URL into the prompt and then press Enter.
+1. When directed to, either accept the default project name or enter a name for your debugging project, then press Enter.
+1. Choose a location to store the project, and then click **Create Project**.
+1. Wait for the project generation process to finish. VS Code creates a Salesforce DX project that contains your packaged metadata, your subscriber's metadata, and skeleton classes for other packages in the org. VS Code also creates a launch configuration (`launch.json` file) for the project. This process can take a long time, especially for orgs that contain a lot of metadata, so feel free to leave it running and check back later. When the project is ready, VS Code opens it for you in a new window.
```
Comment (vazexqi) [inline packages/salesforcedx-vscode-apex-debugger/README.md]: Might be useful to point out that you can monitor the progress in the output panel at the bottom of VS Code. To show the output panel, use View > Output and select Salesforce DX CLI from the pull down menu at the corner of that tab.

screen shot 2018-05-17 at 1 26 45 pm

**Comment by vazexqi** [comment]
Let's not merge this until we have enabled the ISV debugger. Right now, it's still hidden under a pref in  VS Code. We can coordinate that with a proper release date.

File: packages/salesforcedx-vscode-apex-debugger/README.md (hunk 4 above)
Comment (ruthemmanuelle) [reply packages/salesforcedx-vscode-apex-debugger/README.md]: Great idea. Will do.

**Pull request merged**